README.md

```diff
@@ -44,9 +47,24 @@ Make sure you have the necessary packages installed
     conda install -c conda-forge numpy scipy cython joblib threadpoolctl pytest compilers llvm-openmp
```

Comment:
Here could redundant packages like `numpy scipy cython` from pip install be substituted with this conda install? I think this could actually cause env conflict if you pip install a lot when you can install by conda install.